lib/src/main/java/org/team100/lib/motor/drive/Falcon6DriveMotor.java

```diff
@@ -60,10 +60,8 @@ public class Falcon6DriveMotor implements MotorWithEncoder100<Distance100> {
     private static final double accelFFVoltS2_M = 0;
 
     /**
-     * Proportional feedback coefficient for the controller. The error is measured
-     * in sensor units (ticks per 100ms), and the full scale output is 1023.
      */
-    private static final double outboardP = 0.05;
+    private static final double outboardP = .001;
```

Comment:
it doesn't really make sense that the P value should be less if the error value is also less.  it seems like you're just effectively turning off the feedback controller.   maybe add a TODO here to figure it out?

Reply:
Done